common-lib/src/cmr/common/generics_documentation.clj

```diff
@@ -0,0 +1,126 @@
+
+(ns cmr.common.generics-documentation
+  "Defines utilities for new generic document pipeline. Most functions will deal
+   with either returning generic config files, or lists of approved generics."
+  (:require
+   [cheshire.core :as json]
+   [clojure.java.io :as io]
+   [clojure.set :as set]
```

Comment:
rename clojure.set to something else like sset.  clojure.core also has a set function.